lib/features/auth/cubit/auth_cubit.dart

```diff
@@ -47,11 +53,16 @@ class AuthCubit extends Cubit<AuthState> {
         notificationId: newNotificationId,
       );
 
+      final biometricSetting = await BiometricsHelper.getBiometricSetting();
+
       emit(
         state.copyWith(
-          status: AuthStatus.authenticated,
+          status: biometricSetting == BiometricSetting.unknown
+              ? AuthStatus.biometricCheck
+              : AuthStatus.authenticated,
           user: userExt,
           session: session,
+          navigate: navigate,
         ),
       );
```

Comment:
Styling opinion but i would wrap the emit in a guarded if statement like 
If (biometricSetting == BiometricSetting.unknown){
emit(
        state.copyWith(
          status: AuthStatus.biometricCheck,
          user: userExt,
          session: session,
          navigate: navigate,
        ),
      );
      
      return;
      }
      
      I think its easier to follow, but its just an opinion 🦐

Reply:
perhaps, but in this case we will have two emits, so not sure what is better)